frontend/simple/main.js

```diff
@@ -8,7 +9,7 @@ import UserProfileView from './views/UserProfileView.vue'
 import PayGroupView from './views/PayGroupView.vue'
 import NavBar from './views/NavBar.vue'
 import utils, { wrap, lazyLoadVue, superagentHeader } from './js/utils'
-import store from './js/state'
+import store from './js/state.js'
```

Comment:
This `.js` is isn't necessary as far as I know and makes the imports inconsistent (e.g. `'./js/translations'` doesn't have a `.js` either).